tests/test_constrained.py

```diff
@@ -21,7 +21,7 @@
 from symfit import (
     Symbol, MatrixSymbol, Inverse, CallableModel, sqrt, Sum, Idx, symbols
 )
-from tests.test_minimizers import subclasses
+from tests.rec_subclass import subclasses
```

Comment:
This import (and others) should probably be a relative import: `from .rec_subclass import subclasses`. And personally I'd advocate renaming `rec_subclass.py` to e.g. `helper_functions.py`.